src/functions/components/FunctionContextMenu.tsx

```diff
@@ -9,18 +9,23 @@ import {ButtonShape, ComponentColor, IconFont} from '@influxdata/clockface'
 
 // Utils
 import {getOrg} from 'src/organizations/selectors'
+import {FunctionListContext} from 'src/functions/context/function.list'
 
 interface Props {
   id: string
   name: string
 }
 
 const FunctionContextMenu: FC<Props> = ({id, name}) => {
+  const {remove} = useContext(FunctionListContext)
+
   const {id: orgID} = useSelector(getOrg)
 
   const history = useHistory()
 
-  const handleDelete = () => {}
+  const handleDelete = () => {
+    remove(id)
```

Comment:
man.. that const {id} = useSelector(getOrg) gave me heart palpitations when i hit this remove. not used to the renaming syntax yet. all good though.

Reply:
I'll call it functionID :)